docs/manual/layout/templates/template-assets.de.md

```diff
@@ -3,8 +3,7 @@ title: "CSS- und JavaScript-Assets"
 description: "CSS- und JavaScript-Assets in Templates einsetzen."
 url: "layout/templates/assets"
 aliases:
-    - /de/templates/assets/
```

Comment:
This deletes an alias. Was the article never linked on that alias?

Reply:
No, there wass only one file within /de/theme-manager/templates-verwalten/ before.